microhapulator/workflows/analysis.smk

```diff
@@ -211,7 +211,9 @@ rule apply_filters:
     params:
         static=f"--static {config['thresh_static']}",
         dynamic=f"--dynamic {config['thresh_dynamic']}",
-        threshfile="" if config["thresh_file"] == "" else f"--config {config['thresh_file']}",
+        threshfile=(
+            "" if config["thresh_file"] in ("", None) else f"--config {config['thresh_file']}"
+        ),
```

Comment:
I think the newer versions of Snakemake were converting the empty string in the config dictionary into `None`, so this change just accounts for both.